src/containers/AccessPointDetails/components/General/index.js

```diff
@@ -270,6 +271,17 @@ const General = ({
     );
   };
 
+  const renderExtraFields = name => {
+    const item = extraFields.find(i => i.label === name);
```

Comment:
no, i want no knowledge of what the extraFields are in here, just extraFields.map returning the renderItem